server/src/Packages/Paypal/Resources/js/paypal.ts

```diff
@@ -0,0 +1,66 @@
+import $ from 'jquery';
+import successTemplate from './templates/successTemplate';
+import errorTemplate from './templates/errorTemplate';
+
+class PaypalService {
+    constructor() {}
+
+    public onApprove(data: { orderID: number }, actions: {}) {
+        let loading = document.getElementById('loadingDiv');
+        if (loading) loading.style.display = 'flex';
+        return fetch('/create-paypal-transaction', {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/json',
+            },
+            body: JSON.stringify({
+                orderID: data.orderID,
+            }),
+        })
+            .then(function (res) {
+                try {
+                    return res.json();
+                } catch (error) {
+                    console.log(`error: ${error}`);
+                    console.log('error text:' + res.text());
+                }
+            })
+            .then(function (dataRes) {
+                if (dataRes.error) {
+                    console.log(dataRes.error);
+                    return;
+                }
+
+                if (loading) loading.style.display = 'none';
+                $('#paypal-button-container').html(successTemplate('Payment successful'));
+            })
+            .catch((error) => {
+                console.log(`.catch ${error}`);
+            });
+    }
+
+    public loadScript(url: string, callback: any) {
+        var script: any = document.createElement('script');
```

Comment:
Let's replace any use of `var` with `const`, and `let` if the value needs to mutate.

Reply:
cool